includes/Context.php

```diff
@@ -197,6 +197,20 @@ public function get_reference_permalink( $post = 0 ) {
 		return $permalink;
 	}
 
+	/**
+	 * Gets the canonical url for the current request.
+	 */
```

Comment:
Missing return type doc.

Reply:
Added in https://github.com/google/site-kit-wp/pull/543/commits/ba8541758ae39e27bd0abb23085d91a31ed9db74